src/main/java/com/sarality/list/impl/SecondaryActionListitemRender.java

```diff
@@ -0,0 +1,52 @@
+package com.sarality.list.impl;
+
+import android.view.View;
+
+import com.sarality.list.R;
+
+/**
+ * Implementation of the List View Item Renderer where each item has a secondary action
+ *
+ * @author satya (satya puniani)
+ */
+
+public abstract class SecondaryActionListitemRender<T> extends SimpleListItemRender<T> {
```

Comment:
I'll try and refactor these later to expose different ViewHolders for different types of list. For now let's just return common and have different methods to define Actions on various views